scrape/target.go

```diff
@@ -341,42 +340,20 @@ func (app *timeLimitAppender) AddFast(ref uint64, t int64, v float64) error {
 	return err
 }
 
-// populateLabels builds a label set from the given label set and scrape configuration.
-// It returns a label set before relabeling was applied as the second return value.
-// Returns the original discovered label set found before relabelling was applied if the target is dropped during relabeling.
-func populateLabels(lset labels.Labels, cfg *config.ScrapeConfig) (res, orig labels.Labels, err error) {
-	// Copy labels into the labelset for the target if they are not set already.
-	scrapeLabels := []labels.Label{
-		{Name: model.JobLabel, Value: cfg.JobName},
-		{Name: model.MetricsPathLabel, Value: cfg.MetricsPath},
-		{Name: model.SchemeLabel, Value: cfg.Scheme},
-	}
-	lb := labels.NewBuilder(lset)
-
-	for _, l := range scrapeLabels {
-		if lv := lset.Get(l.Name); lv == "" {
-			lb.Set(l.Name, l.Value)
-		}
-	}
-	// Encode scrape query parameters as labels.
-	for k, v := range cfg.Params {
-		if len(v) > 0 {
-			lb.Set(model.ParamLabelPrefix+k, v[0])
-		}
-	}
-
-	preRelabelLabels := lb.Labels()
-	lset = relabel.Process(preRelabelLabels, cfg.RelabelConfigs...)
+// runRelabeling gets the original label set and runs relabeling against it.
```

Comment:
takes, not gets - it's not fetching the labels.

apply would also be better. Run implies that we're starting a service.